src/components/Accordion/Accordion/__tests__/accordion.interactions.ts

```diff
@@ -1,7 +1,8 @@
 import { expect } from "@storybook/jest";
 import { userEvent, within } from "@storybook/testing-library";
 import { Screen } from "@testing-library/react";
-import { delay, interactionSuite, resetFocus } from "../../../../__tests__/interactions-helper";
+import { resetFocus } from "../../../../__tests__/interactions-helper";
```

Comment:
reset focus is something which is super specific to vibe (it using a storybook decorator).
At first I thought not to put it as part of anything we exporting but maybe I will try to find another solution for reset focus without a storybook decorator ? because it is something which will probably be needed.

Reply:
wdyt?